src/main/java/no/difi/asic/AsicUtils.java

```diff
@@ -1,19 +1,23 @@
 package no.difi.asic;
 
 import com.google.common.io.ByteStreams;
+import org.apache.tika.Tika;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.*;
-import java.net.URLConnection;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.regex.Pattern;
 import java.util.zip.ZipEntry;
 
 public class AsicUtils {
 
-    private static Logger logger = LoggerFactory.getLogger(AsicUtils.class);
+    private static final Logger logger = LoggerFactory.getLogger(AsicUtils.class);
+
+    private static final String TIKA_FALLBACK_MIME = "application/octet-stream";
+    private static final Tika TIKA = new Tika();
```

Comment:
Er Tika trådsikker slik at den kan brukes i statisk kontekst?

Reply:
Hei. 

Jeg har nå retestet denne saken med fiks-io-klient-java:1.3.3 som igjen benytter commons-asic:0.10.0, og feilen inntreffer fremdeles når appen kjøres i k8s (se stack-trace nedenfor). Det er testet både med JDK 11 og JDK 17. 

Tanken bak denne PRen var å foreta så få endringer som mulig samtidig som testene passerer.

Base-image for Java 11: adoptopenjdk/openjdk11:jre-11.0.11_9-alpine
Base-image for Java 17: bellsoft/liberica-openjdk-alpine:17

13:39:06.853 [pool-7-thread-8]
                INFO  no.difi.asic.AsicUtils - Unable to determine MIME type using Files.probeContentType(), trying URLConnection.getFileNameMap()
Exception in thread "pool-7-thread-8" java.lang.RuntimeException: java.lang.IllegalStateException: Unable to determine MIME type of validation_report.json
	at no.ks.fiks.io.asice.write.EncryptedAsicWriterImpl.lambda$createAndEncrypt$2(EncryptedAsicWriterImpl.java:79)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: Unable to determine MIME type of validation_report.json
	at no.difi.asic.AsicUtils.detectMime(AsicUtils.java:134)
	at no.difi.asic.AbstractAsicWriter.add(AbstractAsicWriter.java:80)
	at no.ks.fiks.io.asice.write.EncryptedAsicWriterImpl.write(EncryptedAsicWriterImpl.java:106)
	at no.ks.fiks.io.asice.write.EncryptedAsicWriterImpl.lambda$createAndEncrypt$1(EncryptedAsicWriterImpl.java:72)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at no.ks.fiks.io.asice.write.EncryptedAsicWriterImpl.lambda$createAndEncrypt$2(EncryptedAsicWriterImpl.java:72)
	... 3 more